doc/developer-guide.md

```diff
@@ -44,6 +44,10 @@ You can also load tests in any supported browser, which is helpful for debugging
 4. [Integration Tests](../test/integration/rules/)
 5. There are additional tests located in [test/integration/full/](../test/integration/full/) for tests that need to be run against their own document.
 
+### API Reference
+
+[See API exposed on aXe](./API.md#section-2-api-reference)
```

Comment:
Should this say "full development API" to differentiate from the pre-existing API.md?

Reply:
I would change this, but this only leads on the API reference in api.md file, from where, the full reference is a sub section. So leaving this as is.